fs/fs.go

```diff
@@ -17,25 +17,20 @@ func HomeFolder() string {
 	return u.HomeDir
 }
 
-func CreateHomeConfigFolder(folder string) string {
-	u, err := user.Current()
-	if err != nil {
-		panic(err)
-	}
-	path := path.Join(u.HomeDir, folder)
-	if exists, _ := Exists(path); !exists {
-		if err := os.MkdirAll(path, 0740); err != nil {
-			panic(err)
-		}
-	}
-	return path
-}
+// CreateSecureFolder checks if the folder exists and has the appropriate permission rights. If not it creates it.
```

Comment:
PLease add a comment saying it returns an empty string if there is an error with the permissions of an already existing folder.